Reply/app/src/main/java/com/example/reply/ui/ReplyApp.kt

```diff
@@ -49,33 +65,209 @@ fun ReplyApp(
+       NavigationBarItem(
+           selected = true,
+           onClick = { /*TODO*/ },
+           icon = { Icon(imageVector = Icons.Default.Inbox, contentDescription = stringResource(id = R.string.tab_inbox)) }
+       )
+        NavigationBarItem(
+            selected = false,
+            onClick = { /*TODO*/ },
+            icon = { Icon(imageVector = Icons.Default.Article, contentDescription = stringResource(id = R.string.tab_inbox)) }
+        )
+        NavigationBarItem(
+            selected = false,
+            onClick = { /*TODO*/ },
+            icon = { Icon(imageVector = Icons.Outlined.Chat, contentDescription = stringResource(id = R.string.tab_inbox)) }
+        )
+        NavigationBarItem(
+            selected = false,
+            onClick = { /*TODO*/ },
+            icon = { Icon(imageVector = Icons.Outlined.Videocam, contentDescription = stringResource(id = R.string.tab_inbox)) }
+        )
+    }
+}
+
+@Composable
+fun NavigationDrawerContent(
+    selectedDestination: String,
+    modifier: Modifier = Modifier,
+    onDrawerClicked: () -> Unit = {}
+) {
+    Column(
+        modifier
+            .wrapContentWidth()
+            .fillMaxHeight()
+            .background(MaterialTheme.colorScheme.inverseOnSurface)
+            .padding(24.dp)
+    ) {
+        Row(
+            modifier = modifier
+                .fillMaxWidth()
+                .padding(16.dp),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            Text(
+                text = stringResource(id = R.string.app_name).uppercase(),
+                style = MaterialTheme.typography.titleMedium,
+                color = MaterialTheme.colorScheme.primary
+            )
+            IconButton(onClick = onDrawerClicked) {
+                Icon(
+                    imageVector = Icons.Default.MenuOpen,
+                    contentDescription = stringResource(id = R.string.navigation_drawer)
+                )
             }
         }
+        DrawerItem(
+            selected = selectedDestination == ReplyDestinations.INBOX,
+            imageVector = Icons.Default.Inbox,
+            title = stringResource(id = R.string.tab_inbox)
+        )
+        DrawerItem(
+            selected = selectedDestination == ReplyDestinations.ARTICLES,
+            imageVector = Icons.Default.Article,
+            title = stringResource(id = R.string.tab_article)
+        )
+        DrawerItem(
+            selected = selectedDestination == ReplyDestinations.DM,
+            imageVector = Icons.Default.Chat,
+            title = stringResource(id = R.string.tab_dm)
+        )
+        DrawerItem(
+            selected = selectedDestination == ReplyDestinations.GROUPS,
+            imageVector = Icons.Default.Videocam,
+            title = stringResource(id = R.string.tab_groups)
+        )
     }
 }
+
+@Composable
+fun DrawerItem(
```

Comment:
Why not use https://developer.android.com/reference/kotlin/androidx/compose/material3/package-summary#navigationdraweritem?

Reply:
Hmm not sure what was the reason let me check again.